subsys/net/l2/openthread/openthread.c

```diff
@@ -105,12 +152,12 @@ void otPlatRadioGetIeeeEui64(otInstance *instance, uint8_t *ieee_eui64)
 
 void otTaskletsSignalPending(otInstance *instance)
 {
-	k_sem_give(&ot_sem);
+	ot_signal_set(OPENTHREAD_SIGNAL_PENDING, 0);
```

Comment:
Could we settle on how to handle events in Zephyr? The rework by @MarekPorwisz took a completely different approach (and is already on am upstream PR), I had an impression it's all set already.

Reply:
yes, we must discuss it

Reply:
Signals from openthread .c .h have been removed, as we agreed on the meeting. (come back to mutex)